src/components/PeopleList/models/people-list-types.ts

```diff
@@ -0,0 +1,20 @@
+import { PeopleCard } from '../../../global-models';
+
+export interface PeopleListState {
+
```

Comment:
@vadym9 i dont understand this empty interfaces. It's not how it supposed to be for sure.